floodviz/static/js/hydrograph.js

```diff
@@ -18,25 +18,27 @@ document.addEventListener("DOMContentLoaded", function(event) {
 			// might want to switch to lineChart(). When cumulative line chart is clicked, the Y axis freaks out and reverts back to issue at https://github.com/novus/nvd3/issues/695 # 
 			var chart = nv.models.cumulativeLineChart()
 					.x( getX )  // this value is stored in milliseconds since epoch (converted in data_format.py with datetime)
-					.y( getY ) 
-					.color(d3.scale.category10().range())
-					.useInteractiveGuideline(true)
+					.y( getY )
+					.color(["#0000FF"])
+					.useVoronoi(true)
+					.clipVoronoi(false)
+					.useInteractiveGuideline(false)
 					.yScale(d3.scale.log())	// Logarithmic Y axis. (GM - I think this make the flood event less decipherable in the visual but it probably scales better)
 					.margin({left: 120, top: 60})
 					.showLegend(false)
 					.showControls(false);
-
-			chart.xAxis.axisLabel(" Date (M-D-Y)")
+			
+			chart.xAxis.axisLabel(" Date (M-D-Y H:M:S)")
 					.axisLabelDistance(10)
 					.ticks(5)
-					.tickFormat(function(d){return d3.time.format('%m-%d-%y')(new Date(d))});
+					.tickFormat(function(d){return d3.time.format('%m-%d-%y %H:%M:%S %Z')(new Date(d))});
```

Comment:
The ticket does ask for time and timezone in the tooltip, but I can also remove this if that's better. Maybe make it so time is in tooltip, but only date is on x-axis?